README.md

```diff
@@ -21,9 +21,9 @@ bower install chart.js --save
 
 #### Selecting the Correct Build
 
-Chart.js provides two different builds that are available for your use. The `Chart.js` and `Chart.min.js` files include Chart.js and the accompanying color parsing library. If this version is used and you require the use of the time axis, [Moment.js](http://momentjs.com/) will need to be included before Chart.js.
+Chart.js provides two builds: `Chart.js`, `Chart.min.js`. Both `Chart.js` and `Chart.min.js` files include Chart.js as well as the color parsing library. If this version is used, you require to include [Moment.js](http://momentjs.com/) before Chart.js for the functionality of the time axis.
```

Comment:
I would change the second sentence to "If this version is used, you are required to include Moment.js before Chart.js to enable the functionality of the time axis."

Reply:
Hello @etimberg and nice to meet you! Your suggested version is better. Fix in befc611f2b589082e491ed94a986988d083490f3. Thank you.